File: client/src/app/shared/models/review.ts
export interface Review {
  id: string;
  message: string;
  title: string;
  rating: number;
  movieId: string;
  userId: number;
  username: string;
  createdAt: Date;
}

export interface ReviewResponse {
  page: number;
  results: Review[];
  total_results: number;
  total_pages: number;
}

File: server/src/main/java/ibf2022/batch2/miniproject/server/models/Review.java
package ibf2022.batch2.miniproject.server.models;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Review {
   private String id;
   private String message;
   private String title;
   private Integer rating;
   private String movieId;
   private Long userId;
   private String username;
   private LocalDateTime createdAt;
   private LocalDateTime updatedAt;
   private Long createdBy;
   private Long updatedBy;

}
